src/main/cpp/input/InputHandler.h

```diff
@@ -165,6 +167,8 @@ namespace nova {
 		struct mouse_button_event dequeue_mouse_button_event();
 		void queue_mouse_position_event(struct mouse_position_event  e);
 		struct mouse_position_event dequeue_mouse_position_event();
+        void queue_mouse_scroll_event(struct mouse_scroll_event  e);
```

Comment:
Nope! http://stackoverflow.com/questions/15181765/passing-structs-to-functions

In C, maybe